src/libraries/System.Private.CoreLib/src/System/Globalization/OrdinalCasing.Icu.cs

```diff
@@ -40,40 +41,44 @@ internal static partial class OrdinalCasing
         // s_casingTable is covering the Unicode BMP plane only. Surrogate casing is handled separately.
         // Every cell in the table is covering the casing of 256 characters in the BMP.
         // Every cell is array of 512 character for uppercasing mapping.
-        private static ushort []?[] s_casingTable =
+        private static ushort []?[]? s_casingTable;
+
+        // 0 - null
+        // 1 - s_noCasingPage
+        private static byte[] s_casingTableInit =
```

Comment:
Possible optimization: this needn't be a static field. Instead, it could be a property that returns a `ROS<byte>` mapped around the assembly's .data segment. The only consumer of this member doesn't rely on it being a true array.